src/main/java/org/codehaus/mojo/build/CreateTimestampMojo.java

```diff
@@ -1,4 +1,4 @@
-package org.codehaus.mojo.build;
+packag org.codehaus.mojo.build;
```

Comment:
??

Reply:
Oops 😅 how did that happen? Will fix it right away